dist/ember-charts.js

```diff
@@ -3759,13 +3759,37 @@ define('ember-charts/components/vertical-bar-chart', ['exports', 'module', 'embe
       return d3.scale.linear().domain(this.get('yDomain')).range([this.get('graphicTop') + this.get('graphicHeight'), this.get('graphicTop')]).nice(this.get('numYTicks'));
     }),
 
-    individualBarLabels: _Ember['default'].computed('groupedData.[]', function () {
+    groupedIndividualBarLabels: _Ember['default'].computed('groupedData.[]', function () {
       var groups = _.map(_.values(this.get('groupedData')), function (g) {
         return _.pluck(g, 'label');
       });
       return _.uniq(_.flatten(groups));
     }),
 
+    ungroupedIndividualBarLabels: _Ember['default'].computed('sortedData.@each.label', function () {
+      return _.map(this.get('sortedData'), 'label');
+    }),
+
+    // The labels of the bars in the chart.
+    //
+    // When the bars in the chart are grouped, this CP returns the de-duplicated
+    // set of labels that can appear within a single group,
+    // in the order that they should appear in the group.
```

Comment:
May be worth specifying that "the order that they should appear in the group" is just alphabetical, because we do not currently support sorting grouped data with the `sortKey` in any fashion